packages/jest-globals/src/index.ts

```diff
@@ -26,6 +32,30 @@ export declare const beforeEach: Global.GlobalAdditions['beforeEach'];
 export declare const afterEach: Global.GlobalAdditions['afterEach'];
 export declare const afterAll: Global.GlobalAdditions['afterAll'];
 
+declare const jest: Jest;
+
+// eslint-disable-next-line @typescript-eslint/no-namespace
+declare namespace jest {
```

Comment:
I love that we're finally adding type-only `jest.*` exports now! 👍

Hopefully we'll be able to replace most of `@types/jest` soonish with this approach